modules/gapi/src/compiler/gcompiler.cpp

```diff
@@ -438,8 +473,37 @@ cv::GStreamingCompiled cv::gimpl::GCompiler::compileStreaming()
 {
     // FIXME: self-note to DM: now keep these compile()/compileStreaming() in sync!
     std::unique_ptr<ade::Graph> pG = generateGraph();
-    GModel::Graph(*pG).metadata().set(Streaming{});
+    GModel::Graph gm(*pG);
+    gm.metadata().set(Streaming{});
```

Comment:
Is it necessary to split one string into two different? As I can see, `gm` object is never used after..

Reply:
+1